src/index.js

```diff
@@ -105,42 +111,56 @@ function escapeChars(message) {
 }
 
 /**
- * Parse reviews of PR into symbolic form described as:
+ * Return emoji for review state
  *
  * ✅ approved
  * ❌ changes requested
  * 💬 commented
  * 🔸 review is pending
  *
+ * @param {string} state - review state
+ * @returns {string}
+ */
+function getReviewStateEmoji(state = '') {
+  switch (state) {
+    case 'APPROVED': return '✅';
+    case 'CHANGES_REQUESTED': return '❌';
+    case 'COMMENTED': return '💬';
+    default: return '🔸';
+  }
+}
+
+/**
+ * Parse reviews of PR into symbolic form
+ *
  * @param {Array} latestOpinionatedReviews - list of latest opinionated reviews on PR
  * @param {Array} latestReviews - list of lastest reviews on PR
  * @param {Array} reviewRequests -  list of review requests on PR
  * @returns {string} - Symbolic string Contains parsed form of reviews
  */
 function createReviewStatus(latestOpinionatedReviews, latestReviews, reviewRequests) {
-  let reviewStatus = '';
-
-  latestReviews = latestReviews.nodes;
+  const reviewReport = {};
 
   latestOpinionatedReviews.nodes.forEach(({ state, author }) => {
     const person = author.login;
 
-    reviewStatus += state === 'COMMENTED' ? '💬' : '';
-    reviewStatus += state === 'APPROVED' ? '✅' : '';
-    reviewStatus += state === 'CHANGES_REQUESTED' ? '❌' : '';
-    latestReviews = latestReviews.filter((item) => {
-      return person !== item.author.login;
-    });
+    reviewReport[person] = getReviewStateEmoji(state);
   });
 
-  latestReviews.forEach(({ state }) => {
-    reviewStatus += state === 'COMMENTED' ? '💬' : '';
-    reviewStatus += state === 'APPROVED' ? '✅' : '';
-    reviewStatus += state === 'CHANGES_REQUESTED' ? '❌' : '';
+  latestReviews.nodes.forEach(({ state, author }) => {
```

Comment:
I guess `latestReview` come first and then `latestOpiniontedReview`. As well as we need to check how latest review comes in order because we are selecting last which means it comes in descending order of review.which means you will get the reviews of in between not latest